sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -20,6 +19,15 @@
 from feast.registry import Registry
 from feast.repo_config import RepoConfig
 
+try:
+    from google.auth.exceptions import DefaultCredentialsError
+    from google.cloud import bigquery
```

Comment:
here as well, I think (though it may have been just one of gcp.py and bigquery.py where this happened)